### PR TITLE
Convert to topkg.

### PR DESCRIPTION
**Pull request by yomimono**
I removed some dependencies in `opam` which `topkg lint` noted weren't used in `_tags` anywhere.  They seem to be required in any situation where you'd actually use `mirage-qubes` but not for the project itself.

The added `LICENSE.md` added contains the text from https://opensource.org/licenses/BSD-2-Clause , as `BSD-2-Clause` is listed as the license in the `opam` file.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Copyright (c) 2015 Thomas Leonard
```
Comment (talex5) [inline LICENSE.md]: Maybe remove this from the README then, to avoid duplication?

Comment (yomimono) [inline LICENSE.md]: Ah, missed that it was there, thanks! (banner blindness but for licenses, I guess)

**Comment by yomimono** [comment]
I realized those dependencies were needed to get the `vchan.xen` depopt and added them back in in https://github.com/talex5/mirage-qubes/pull/1/commits/cae1fd1e912535373db96f682cd703b773314b4c , sorry for the noise.

**Comment by talex5** [comment]
I just triggered a rebuild of master, and that failed too. Looks like something changed upstream...

**Comment by yomimono** [comment]
I noticed this vchan build failure a while ago and filed an issue, but it doesn't look like anyone's gotten around to looking at it yet: https://travis-ci.org/mirage/mirage/jobs/167401772#L5506

**Comment by yomimono** [comment]
@talex5 is there a reason for the unusual .travis.yml here?  I'm happy to do a bit of debugging around the new test failures, but I'd rather not rediscover problems you've already worked around.

**Comment by talex5** [comment]
No, I think it's just old and should probably be replaced with a modern version.

**Comment by yomimono** [comment]
4.04 tests are failing because mirage-xen-ocaml is version-constrained to < 4.04 in the default opam remote.  There's a newer version in `mirage/mirage-dev`, but it's mirage-3 only and I'm not sure whether you want to enter that universe yet.  Tests are now passing for 4.02 and 4.03, though.

**Comment by talex5** [comment]
Can we just remove the `OCAML_VERSION=4.04` for now?

**Comment by yomimono** [comment]
Sure, no problem.

**Pull request merged**